src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -22,7 +22,10 @@ import { ExportDialogDeps } from 'DetailsView/components/export-dialog';
 import { NarrowModeStatus } from 'DetailsView/components/narrow-mode-detector';
 import { ReportExportButton } from 'DetailsView/components/report-export-button';
 import { ReportExportDialogFactoryProps } from 'DetailsView/components/report-export-dialog-factory';
-import { SaveAssessmentButton } from 'DetailsView/components/save-assessment-button';
+import {
+    SaveAssessmentFactoryDeps,
+    SaveAssessmentFactoryProps,
+} from 'DetailsView/components/save-assessment-factory';
```

Comment:
This PR is including your changes from #3617. You may want to pull the latest changes into your fork's master branch and merge those changes into this branch.